server/service/src/sync/init_programs_data.rs

```diff
@@ -457,6 +458,48 @@ fn encounter_hiv_care_5() -> hiv_care_encounter::HivcareEncounter {
     })
 }
 
+pub fn insert_programs_permissions(connection: &StorageConnection, user_id: String) {
+    UserPermissionRowRepository::new(&connection)
+        .upsert_one(&UserPermissionRow {
+            id: uuid(),
+            user_id: user_id.clone(),
+            store_id: Some("D77F67339BF8400886D009178F4962E1".to_string()),
+            permission: Permission::DocumentProgramQuery,
+            context: Some("HIVCareProgram".to_string()),
+        })
+        .unwrap();
+
+    UserPermissionRowRepository::new(&connection)
+        .upsert_one(&UserPermissionRow {
+            id: uuid(),
+            user_id: user_id.clone(),
+            store_id: Some("D77F67339BF8400886D009178F4962E1".to_string()),
+            permission: Permission::PatientQuery,
+            context: None,
+        })
+        .unwrap();
+
+    UserPermissionRowRepository::new(&connection)
+        .upsert_one(&UserPermissionRow {
+            id: uuid(),
+            user_id: user_id.clone(),
+            store_id: Some("D77F67339BF8400886D009178F4962E1".to_string()),
+            permission: Permission::Document,
+            context: None,
+        })
+        .unwrap();
+
+    UserPermissionRowRepository::new(&connection)
+        .upsert_one(&UserPermissionRow {
+            id: uuid(),
+            user_id,
+            store_id: Some("D77F67339BF8400886D009178F4962E1".to_string()),
+            permission: Permission::DocumentProgramMutate,
+            context: Some("Patient".to_string()),
+        })
+        .unwrap();
+}
+
```

Comment:
I'm being asked for `DocumentEncounterQuery` permission too - this is to view the list of encounters. Do you get that as well? or is the above enough?

Reply:
[b5f8c8b](https://github.com/openmsupply/open-msupply/pull/611/commits/b5f8c8b3f42060c70799ff2fcdad6d3f2107ed42) 

Inserted permissions for all stores since couldn't add store id from the `update_user` 🤔 (or maybe I might have missed something).... more things to be deleted later.... 😰